backend/src/attraction/routes/router.py

```diff
@@ -22,6 +22,15 @@ def get_all_attractions(
     return _attraction_service.get_all_attractions(db)
 
 
+@router.get("")
+def get_attractions(
+    filters: AttractionFilters = Depends(),
```

Comment:
Is that queryparams ?

Reply:
Yes, this is query params

Reply:
Shouldnt we use it like this ? 
https://fastapi.tiangolo.com/tutorial/query-params/